randovania/games/super_metroid/json_data/Norfair.txt

```diff
@@ -692,7 +692,10 @@ Extra - asset_id: 206
   > Door to Cathedral
       All of the following:
           Varia Suit
-          Hi-Jump Boots or Space Jump
+          Any of the following:
+              Hi-Jump Boots or Space Jump
+              # TODO: not familiar, does this need WJ? damage boost?
+              Enabled Nova Boost Tweaks
 
```

Comment:
It needs a WJ, damage boost, and morph ball. Its an incredibly difficult trick. Also separately I would add a Beginner Springball Jump.

https://youtu.be/LzMLxHH4rZQ?si=6H5dt078PanSVXIv

Reply:
Thats for vanilla, not for the patch that redoes room geometry which is what the comment is referring to.

Reply:
I see, I'm not sure if the patch still requires a WJ then either. I don't have it handy. I'll have to roll a seed to check.

Reply:
The patch still requires wall jump. The patch makes it so that its just a simple wall jump.
![image](https://github.com/randovania/randovania/assets/46821362/34161331-b7a4-410f-8958-e121aa5d4d83)